book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,246 @@
+# Platform Tier Policy
+
+## Table of Contents
+
+- [Platform Tier Policy](#platform-tier-policy)
+  - [Table of Contents](#table-of-contents)
+  - [General](#general)
+  - [Tier 3 platform policy](#tier-3-platform-policy)
+  - [Tier 2 platform policy](#tier-2-platform-policy)
+  - [Tier 1 platform policy](#tier-1-platform-policy)
+
+## General
+
+The Zcash Foundation provides three tiers of platform support, modeled after the
+[Rust Target Tier Policy](https://doc.rust-lang.org/stable/rustc/target-tier-policy.html):
+
```

Comment:
"They exist in the codebase" is a Rust compiler-specific note.

These platforms might not even be mentioned in the Zebra codebase.